packages/@aws-cdk/aws-iam/lib/grant.ts

```diff
@@ -101,7 +101,7 @@ export interface GrantOnPrincipalAndResourceOptions extends CommonGrantOptions {
  * This class is not instantiable by consumers on purpose, so that they will be
  * required to call the Grant factory functions.
  */
-export class Grant implements cdk.IDependable {
+export class Grant implements IDependable {
```

Comment:
could we do `export class Grant implements cdk.IDependable, IDependable {` on master, and mark the former as deprecated?